plugin-server/src/types.ts

```diff
@@ -533,22 +554,21 @@ export interface Event {
     group4_properties: Record<string, any>
 }
 
-export interface ClickHouseEvent extends Omit<Event, 'id' | 'elements' | 'elements_hash'> {
-    uuid: string
-    elements_chain: string | undefined
+export interface IngestionEvent {
```

Comment:
Maybe a docstring here too for when / where this should be used instead of RawEvent or Event